uber/models.py

```diff
@@ -615,8 +615,8 @@ def change_badge(self, attendee, badge_type, badge_num=None):
             return 'Badge updated'
 
         def everyone(self):
```

Comment:
Because the idea of the badge statuses is that we only care about New and Complete badges, I changed the `everyone` variable here to reflect that.